fastlane/Fastfile

```diff
@@ -736,9 +735,13 @@ end
   #####################################################################################
   desc "Occupy a free test account and run UI tests with it"
   lane :pick_test_account_and_run_ui_tests do | options |
-    occupy_test_account
-    run_ui_tests(options.merge({ test_account: "simplenoteuitest-ios-#{$used_test_account_index}@a8c.com" }))
-    deoccupy_test_account
+    if is_ci
+      occupy_test_account
+      run_ui_tests(options.merge({ test_account: get_required_env("UI_TESTS_ACCOUNT_EMAIL").sub("X", $used_test_account_index.to_s)}))
+      deoccupy_test_account
+    else
+      UI.message("This lane runs only on CI")
```

Comment:
How do you feel about changing this to a user error?

```suggestion
      UI.user_error!("This lane should be run only from CI")
```

I think a failure would make it clear to a developer calling this lane locally that they're doing something inappropriate. Otherwise, the `fastlane` call will succeed and they might not realize the side effect they expected didn't occur.

Reply:
As an alternative that doesn't fail the lane, this could become:

```suggestion
      UI.important("This lane runs only on CI")
```

This way, the output will be colored and the caller will be more likely to notice it:

![image](https://user-images.githubusercontent.com/1218433/120944954-7a7ecd00-c77a-11eb-8b38-2c59c1d405e1.png)

Reply:
Thank you @mokagio ! Makes sense, I went for the `UI.user_error!`